src/QuadForm/Lattices.jl

```diff
@@ -1135,7 +1135,7 @@ function trace_lattice_with_isometry_and_transfer_data(H::AbstractLat{T}; alpha:
     return H, f, AbstractSpaceRes(V, V, identity_matrix(E, n), identity_matrix(E, n))
   end
 
-  @req (degree(E) == 2) && (is_totally_complex(E)) && (is_totally_real(base_field(E))) "The base field of H must be CM"
```

Comment:
I think this still holds, even in the infinite order case.

Reply:
In the example of the issue it does not hold, `E` is real. It does not seem to be actually required to have `E/K` CM.

Reply:
You are right of course.